src/test/resources/localest.properties

```diff
@@ -0,0 +1,6 @@
+jira.protocol = http
+jira.host = localhost
+jira.port = 2990
+jira.context = /jira
+jira.edition = all
+jira.xml.data.location = src/test/xml
```

Comment:
do you really need this properties

Reply:
this for integration test but  now it is not useful